src/Illuminate/Database/Eloquent/Builder.php

```diff
@@ -1340,9 +1341,7 @@ public static function __callStatic($method, $parameters)
         }
 
         if (! isset(static::$macros[$method])) {
-            throw new BadMethodCallException(sprintf(
-                'Method %s::%s does not exist.', static::class, $method
-            ));
+            $this->throwBadMethodCallException($method);
```

Comment:
I don't know if anyone else caught this, but you can't call `$this` from a static method...